extension/src/popup/components/manageAssets/AddAsset/index.tsx

```diff
@@ -91,17 +92,23 @@ export const AddAsset = ({ setErrorAsset }: AddAssetProps) => {
               </div>
               <div className="AddAsset__results">
                 {isCurrencyNotFound ? (
-                  <InfoBlock>Currency not found</InfoBlock>
+                  <InfoBlock>Asset not found</InfoBlock>
                 ) : null}
                 {assetRows.length ? (
                   <>
                     <div className="AddAsset__title">
                       Assets found in this domain
                     </div>
-                    <ManageAssetRows
-                      assetRows={assetRows}
-                      setErrorAsset={setErrorAsset}
-                    />
+                    <div
+                      className="AddAsset__results__rows"
+                      ref={ManageAssetRowsWrapperRef}
+                    >
+                      <ManageAssetRows
+                        assetRows={assetRows}
+                        setErrorAsset={setErrorAsset}
+                        parentRef={ManageAssetRowsWrapperRef}
```

Comment:
💭  could we just pass in the height as a prop? eg:
```suggestion
                        maxHeight={ManageAssetRowsWrapperRef.current?.clientHeight}
```
it would simplify how `ManageAssetRows` is used (assuming we don't need the parentRef for anything else)

Reply:
Good idea - that's more straightforward